### PR TITLE
Update project structure; Download single file from s3

### PR DESCRIPTION
**Pull request by camenpihor**
fixes issue #9 

## Changes

- updates project structure given new repository name and responsibilities
- adds functionality for downloading a single file from s3

### CODE REVIEW

**Comment by camenpihor** [comment]
Should we test multiple versions of python or windows? @Ferrumofomega 

Looking into adding environment variables to tests so that we can run integration tests too. 
Also looking into adding a badge to master. 
Currently, if we push directly to master these tests will not run. Will look at fixing this.

**Comment by camenpihor** [comment]
To manage notifications about github actions: https://help.github.com/en/github/receiving-notifications-about-activity-on-github/choosing-the-delivery-method-for-your-notifications#choosing-the-delivery-method-for-your-notifications-about-github-actions

**Comment by Ferrumofomega** [comment]
I'm inclined to not test multiple versions of python. Given we make the VM rather easy to work with, and the functionality is rather standalone.

With respect to windows, for now, I am also inclined to not include. How hard would it be to add windows later @camenpihor (my gut says not too bad)?

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def get_version():
 
 
 setup(
-    name="goes",
+    name="wildfire",
     version=get_version(),
     url="https://github.com/Ferrumofomega/goes",
     author="",
```
Comment (Ferrumofomega) [inline setup.py]: Let's add "Camen Pihor", :-).

**Comment by camenpihor** [comment]
Pretty easy! So all good for leaving out right now!

So we currently test on Ubuntu and Mac and only python3.6 (although if we
wanted it would still be pretty easy to pivot to python 3.8)

**Pull request merged**